src/ints/mouse.cpp

```diff
@@ -524,6 +524,7 @@ void Mouse_ButtonPressed(Bit8u button) {
 	case 0:
 		if (mouse.buttons&1) return;
 		mouse.buttons|=1;
+		LOG_MSG("Mouse_AddEvent(MOUSE_LEFT_PRESSED)");
```

Comment:
Should this log always be on?  I can see in some games this could fill up the console window.  One of the things I think could be improved overall are logging controls both in release builds and debug builds.  Easy to lose track of information if there are too many things logging.

Reply:
I agree that it could be changed to the LOG_MOUSE type to reduce log message flooding in such case.